docs/design/caching_scan_results_by_repo_digest.md

```diff
@@ -102,16 +102,33 @@ bar         replicaset-nginx-6d4cf56db6-nginx   library/nginx   1.16   Trivy
 foo         replicaset-nginx-6d4cf56db6-nginx   library/nginx   1.16   Trivy     6m10s
 ```
 
+## Life-cycle management
```

Comment:
Thank you for sharing your thoughts. I assume that it is on purpose that this section is written like a discussion? I would not link PRs in the docs. A better place to post this is in the discussion section on GitHub. The design section of the docs should have definite, maybe even opinionated information.
So once an agreement has been reached in the discussion section, it can translate into a docs section

Reply:
@AnaisUrlichs We are not publishing docs/desing/*.md with official documentation https://aquasecurity.github.io/starboard/

This is internal purpose docs for contributors and maintainers.

Reply:
@NissesSenap I agree this feature is important enough to have it enable by default, probably want to set it up to longer period ~ every 3 days

Reply:
Sure I can update the default value to 3 days. Anything else that you think we should add/change?

Reply:
Nope, all look good !!

Reply:
Oh sorry about that, I thought it is added to the docs. It just sounds more of a discussion to me than a design decision but if this works for everyone then please ignore me :D

Reply:
Update the pr now @chen-keinan